web/urls.py

```
@@ -472,6 +472,22 @@
     path("membership/update-payment-method/", views.update_payment_method, name="update_payment_method"),
     path("membership/update-payment-method/api/", views.update_payment_method_api, name="update_payment_method_api"),
     path("test-sentry-error/", lambda request: 1 / 0, name="test_sentry"),
+    # Virtual Classroom URLs
+    path('virtual-classroom/', views.virtual_classroom_list, name='virtual_classroom_list'),
+    path('virtual-classroom/create/', views.virtual_classroom_create, name='virtual_classroom_create'),
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)